apps/landingPage/components/Headline/GetStartedInfo.js

```diff
@@ -64,28 +84,11 @@ const ContainerLeftText = styled.div`
 
 const WrapperText = styled.div`
   width: 85vw;
+  padding: ${defaultTokens.spaceMedium} 0 ${defaultTokens.spaceMedium} 0;
   @media (min-width: ${BREAKPOINTS.TABLET}px) {
-    width: 25vw;
-  }
-`;
-
-const WrapperButtons = styled.div`
-  display: flex;
-  width: 80vw;
-  flex-direction: column;
-  margin-top: 20px;
-  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}) and (max-width: ${BREAKPOINTS.TABLET}px) {
-    width: 60vw;
-  }
-  @media (min-width: ${BREAKPOINTS.TABLET}px) {
-    width: unset;
-    flex-direction: row;
+    width: 75vw;
   }
-`;
-
-const WrapperRightButton = styled.div`
-  margin-top: 10px;
-  @media (min-width: ${BREAKPOINTS.TABLET}px) {
-    margin: 0 0 0 1vw;
+  @media (min-width: 1025px) {
```

Comment:
What about use some constant here? Btw 1025 is quite interesting number, but I understand that you want style only for bigger screens than 1024px 😄

Reply:
Hmm I really should stop taking numbers from my head 😅
I ve created `BIG_TABLET` media query breakpoint which I ll change to 1024 now
And probably missed this one for changing it to constant